linetools/lists/mk_sets.py

```diff
@@ -0,0 +1,125 @@
+"""
+Module for making set lists
+  INTERNAL USE ONLY
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import os, imp, glob
+
+from astropy import units as u
+from astropy.units.quantity import Quantity
+from astropy import constants as const
+from astropy.io import fits, ascii
+from astropy.table import QTable, Column, Table
+
+from xastropy.xutils import xdebug as xdb
+
+lt_path = imp.find_module('linetools')[1]
+xa_path = imp.find_module('xastropy')[1]
+
+#
+def mk_ism(outfil=None, clobber=False):
+    ''' Make the ISM list from grb.lst in xastropy
```

Comment:
Can we use 'overwrite' instead of 'clobber'? It's less confusing for the uninitiated.